WilliamsGarcia/30-08/HackerRank/ArrayList/ArrayListExample.java

```diff
@@ -0,0 +1,44 @@
+package com.bootcamp.daythree.hackerrank.arraylist;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class ArrayListExample {
+
+    public static final String WHITE_SPACES = "\\s+$";
+    public static final String SPACE = " ";
+    public static final String WITHOUT_SPACE = "";
+    public static final String ERROR = "ERROR!";
+
+    public static void main(String[] args) {
+        Scanner scan = new Scanner(System.in);
+        int arrayNumber = scan.nextInt();
+        scan.nextLine();
+        ArrayList<String[]> auxList = new ArrayList<>(arrayNumber);
```

Comment:
With array string I save the complete input in the array. And is most easy to validate the second part, because my postion 0 always is the number of columns. But I think I can solve it to another form @luisod78